sale_order_pricelist_tax/models/account_move.py

```diff
@@ -0,0 +1,31 @@
+# Copyright (C) 2022 Akretion (<http://www.akretion.com>).
+# @author Kévin Roche <kevin.roche@akretion.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import _, api, models
+from odoo.exceptions import UserError
+
+
+class AccountMove(models.Model):
+    _name = "account.move"
+    _description = "Account Move"
+    _inherit = ["account.move", "price.include.tax.mixin"]
+
+    @api.depends(
+        "invoice_line_ids",
+        "invoice_line_ids.tax_ids",
+        "invoice_line_ids.tax_ids.price_include",
```

Comment:
```suggestion
        "invoice_line_ids.tax_ids.price_include"
```
It's enough as invoice_line_ids, invoice_line_ids.tax_ids will be included